commerce_coordinator/apps/order_fulfillment/serializers.py

```diff
@@ -12,7 +12,4 @@ class FulfillOrderWebhookSerializer(CoordinatorSerializer):
     is_fulfilled = serializers.BooleanField()
     entitlement_uuid = serializers.CharField(required=False, allow_null=True)
     order_id = serializers.CharField()
-    order_version = serializers.CharField()
```

Comment:
Serializer name should be either `FulfilledOrderWebhookSerializer` or `FulfillCompletionOrderWebhookSerializer `